.github/workflows/build-and-test.yml

```diff
@@ -89,7 +89,7 @@ jobs:
       uses: gradle/actions/setup-gradle@94baf225fe0a508e581a564467443d0e2379123b # v4
 
     - name: Run tests
-      run: ./gradlew test -Dkotest.tags='!Integration'
```

Comment:
Note that this runs *all* checks, also non-test ones, like static analysis.

Reply:
Yes. But I still think it's better than calling jvmTest (and possibly also jsTest in the future) separately.

Reply:
According to [these docs](https://www.jetbrains.com/help/kotlin-multiplatform-dev/multiplatform-run-tests.html#work-with-more-complex-projects), there should be an `allTests` task. And I was able to confirm it:

```
$ ./gradlew tasks --all | grep allTest
api:v1:api-v1-client:allTests - Runs the tests for all targets and create aggregated report
api:v1:api-v1-mapping:allTests - Runs the tests for all targets and create aggregated report
api:v1:api-v1-model:allTests - Runs the tests for all targets and create aggregated report
cli:allTests - Runs the tests for all targets and create aggregated report
model:allTests - Runs the tests for all targets and create aggregated report
utils:test:allTests - Runs the tests for all targets and create aggregated report
```

Reply:
`allTests` is only available for multiplatform projects.
For now I'd like to stay with `check`.

Reply:
`allTest` seems to only run tests in the multiplatform projects.
I reverted to check, because this change was just meant to ensure that all tests are run, without the developer having to think about the test source set executed in CI.